images/heapster/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Kubekins 1.6 + gcloud + heapster
+FROM gcr.io/k8s-testimages/kubekins-test:1.6-v20161220-8366258
+MAINTAINER senlu@google.com
+
+RUN apt-get update && apt-get install -y \
+    python \
+    wget && \
+    apt-get clean
+
+ENV GCLOUD_VERSION 142.0.0
+RUN wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-$GCLOUD_VERSION-linux-x86_64.tar.gz && \
```

Comment:
I think other images use `curl` instead, which prevents needing to install `wget`. or do the heapster tests need `wget`?